specification/nhais-adaptor.yaml

```diff
@@ -85,21 +93,40 @@ info:
       Elsewhere in the NHS estate this value may be referred to as the "Partnership Code", "Practice Code", or most
       confusingly the "GP Code". If a practice is assigned different values for any of these then the operator of the
       NHAIS system should be consulted to determine which to use.
+    
+      > This is not the same the as the practice ODS code, and will be provided with by PCSE at the point that the link 
+      is established. This is a Local Code of PCSE’s choosing.
 
       #### GP Trading Partner Code
 
       The GP Trading Partner Code is a four-character (uppercase letters and digits) code used in the interchange header
       of the EDIFACT messages. This may also be referred to as the "GP Link Code".
-
+    
+      > For English Practices, the supplier will be provided with this by PCSE (Primary Care Support England, Capita),
```

Comment:
Not sure about calling out Capita here.

Reply:
removed Capita. I had included as the original email sent with the details was corrected from `PCSE (Primary Care Support England)` - to PCSE (Primary Care Support England, Capita) and the PCSE website is very explicit on it being a capita service!